homeassistant/components/homematicip_cloud/light.py

```diff
@@ -117,3 +124,143 @@ def supported_features(self):
     async def async_turn_off(self, **kwargs):
         """Turn the light off."""
         await self._device.set_dim_level(0)
+
+
+class HomematicipNotificationLight(HomematicipGenericDevice, Light):
+    """Representation of HomematicIP Cloud dimmer light device."""
+
+    _channel_index = None
+
+    # Dictionary to translate between RGBColorState and hs_color
+    _color_switcher = None
```

Comment:
We don't need to initialize this here, since we do that in init.

Reply:
Done